src/raphael_backend_flask/exceptions.py

```diff
@@ -0,0 +1,4 @@
+class FlashException(Exception):
+    def __init__(self, message: str, category: str = "danger") -> None:
```

Comment:
The default category here is "danger", but the only times this is overridden currently, it’s set to "danger" anyway. Probably those overrides could be removed (or this default should be something different).

Also, I guess we should be calling `super().__init__(message)` here.